opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -97,9 +114,16 @@ def collect(
     ) -> Optional[_DataPointVarT]:
         pass
 
+    def _collect_exemplars(self) -> Sequence[Exemplar]:
+        return self._reservoir.collect(
+            self._attributes
+        )  # FIXME provide filtered data point attributes
```

Comment:
Nit: Mark as TODO instead of FIXME.

Is this going to be done as part of this PR or different?